test/distributed/fsdp/test_fsdp_misc.py

```diff
@@ -261,6 +263,151 @@ def _check_equal(local, fsdp):
 
         dist.barrier()
 
+    @skip_if_lt_x_gpu(2)
+    def test_fsdp_optim_overlap_no_use_orig_params_error(self):
+
+        class MyModel(nn.Module):
```

Comment:
nit: This module definition repeated across the two added unit tests could be de-duplicated. This unit test does not really depend on the model specifically, so we could similarly use a common model already defined.